input/pagecontent/conformance.md

```diff
@@ -37,10 +27,12 @@ The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU eR
 
 AU eRequesting profile elements include mandatory and *Must Support* requirements. [Mandatory elements](#mandatory-elements) are required and have a minimum cardinality of 1 (min=1). [Must Support](#must-support-and-obligation) elements have defined conformance obligations in AU eRequesting based on actor roles.
 
+The [Actors and Capabilities](capability-statements.html) page list the AU eRequesting Actor definitions and the CapabilityStatements.
```

Comment:
"the CapabilityStatements" should be 'their CapabilityStatements"

Reply:
Amended, thanks.